common/src/main/java/com/wynntils/features/inventory/ContainerSearchFeature.java

```diff
@@ -212,14 +218,21 @@ private void addSearchWidget(AbstractContainerScreen<?> screen, int renderX, int
         screen.addRenderableWidget(lastSearchWidget);
     }
 
-    private void matchItems(String searchStr, AbstractContainerScreen<?> screen) {
```

Comment:
Do you really have to change the signature of this method? You could just as well just do `screen.getMenu()` inside it, right?

Reply:
because im doing `%9` it makes more sense to "guarantee" a ChestMenu

Reply:
I think it is in more or less always preferable to change as little as possible when fixing a bug, unless that leads to code of too low quality.

Minimal impact of changes is defintely a quality of its own, worthy to pursue in PRs.

So, yes.

Reply:
I'm using ChestMenu#getContainer to only get the items of the actual container which an exclusive method of ChestMenu so all these changes are necessary.